_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "Online Workshop Recipe"
+---
+
+# A Recipe for an Online Carpentries Workshop
+
+In the past year+ of teaching online, our community has learned many things. Our initial [online training recommendations](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop) recognized that we were in uncharted territory for Carpentries workshops! In consulting with community members, we heard many different narratives and suggestions. Rather than creating a single 'recipe,' we chose to distill that advice into 'things to think about', giving Instructors space to engineer their own solutions to the challenges presented by teaching online.
+
+We continue to be excited to learn from the innovations of our Instructor community! However, at this point we have also developed a better idea about the specific advantages and disadvantages of different practices. For those who do not wish to 'reinvent the wheel' but prefer to start with a recommended model, we offer an example structure for an online workshop that we expect to function well.
+
+Note that these recommendations are based characteristics of Zoom, Etherpad, and Slack platforms and may require minor modifications to function on other platforms.
+
+## 1. Advance preparation
+### Workshop roles
+* **Instructors** (at least 2): take the lead on preparing the curriculum and leading instruction. 
+* **Supporting Instructors** (at least 1): take the lead on preparing the workshop mechanics, coordinating with helpers, and ensuring that everything is working smoothly. They may take on limited instruction and/or an instructional backup role if they choose. They play the role of "Zoom DJ" (below) whenever they are not teaching.
+### Schedule
+* A **4 half-day** schedule is ideal where practical. This is  helpful in combatting 'zoom fatigue.'
```

Comment:
Maybe link to a sample 4 day schedule if available

Reply:
Just a side note, 4 days would not work in the South African context, we have found that 5 days are long but leaves the instructors with more room to cover the curriculum and do troubleshooting as well

Reply:
Is this more inclusive?
```suggestion
* A **4 half-day** (or other time-distributed) schedule is ideal where practical. This is  helpful in combatting 'zoom fatigue.'
```

Reply:
Sounds good :)